### ISSUE TITLE
IETF-style RFC

### ISSUE THREAD

**Issue by renne** [ipfs-inactive/RFC#7]
The IETF is the recognized authority when it comes to internet standards.
I suggest to follow the rules/fomats of [IETF internet drafts]( IETF) to make publication via IETF working groups possible.

**Comment by Stebalien** [comment]
TL;DR: These are not the RFCs you're looking for. RFC is a massively overloaded term (originally meaning "request for comments").

This RFC system we use was created by the Rust programming language developers and has a significantly different goal than the IETF RFC system. The IETF's system is great for publishing detailed specs however, it has never (as far as I know) really worked well for higher level design and *decision making*. That is, a large part of *this* RFC process centers around motivation and design rather than implementation details (which are often an unnecessary burden until the tough design decisions have been made).

Many specs that come out of these RFCs *will* likely be turned into IETF RFCs. However, the IETF RFCs will likely need to be written from scratch with detailed design information.
